webpack.config.develop.js

```diff
@@ -0,0 +1,33 @@
+const commonConfig = require('./webpack.config.common');
+const webpackMerge = require('webpack-merge');
+const webpack = require('webpack');
+
+module.exports = webpackMerge(commonConfig, {
+
+  output: {
+    filename: '[name].bundle.js'
+  },
+
+  module: {
```

Comment:
can this `module` property be omitted if it's just providing an empty `rules` value?